### PR TITLE
Added TravisCI and CodeCov hook

### PR DESCRIPTION
**Pull request by FrenchBen**
Added CodeCov hook - Will need to setup the `CODECOV_TOKEN` in CircleCI to see this pass.

Signed-off-by: French Ben <frenchben@docker.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# p9p [![GoDoc](https://godoc.org/github.com/docker/go-p9p?status.svg)](https://godoc.org/github.com/docker/go-p9p) [![CircleCI](https://circleci.com/gh/docker/go-p9p.svg?style=shield)](https://circleci.com/gh/docker/go-p9p) [![Go Report Card](https://goreportcard.com/badge/github.com/docker/go-p9p)](https://goreportcard.com/report/github.com/docker/go-p9p)
+# p9p [![GoDoc](https://godoc.org/github.com/docker/go-p9p?status.svg)](https://godoc.org/github.com/docker/go-p9p) [![CircleCI](https://circleci.com/gh/docker/go-p9p.svg?style=shield)](https://circleci.com/gh/docker/go-p9p)
+[![TravisCI](https://travis-ci.org/docker/go-p9p.svg?branch=codecov)](https://travis-ci.org/docker/go-p9p)
```
Comment (stevvooe) [inline README.md]: These need to stay up on the title line. Otherwise, we won't have enough badges.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+language: go
+
+go:
+  - 1.6
+
+script:
+  - go test -coverprofile=coverage.txt -covermode=atomic
```
Comment (stevvooe) [inline .travis.yml]: Throw a `-race` on here, while we're at it.

**Comment by FrenchBen** [comment]
License badge added as: [Apache licensed](https://raw.githubusercontent.com/docker/go-p9p/master/LICENSE)

**Comment by stevvooe** [comment]
Do you even badge, bro?

LGTM

**Pull request merged**